### DISCUSSION TITLE
Aggregate field column in Content Tables

### DISCUSSION THREAD

**Discussion by tenebrius** [directus/directus#15431] [Feature Requests]
How about the possibility to show aggregate fields in content tables?
For example, if one has a collection 'Course' and m2o relation of 'lessons'. If I want to show the count of favorites for each course on the content tables, an aggregate column would do the trick

Aggregate columns already exist in filters 
![image](https://user-images.githubusercontent.com/3369571/188686521-7e77b596-a68c-4f1e-ab80-3c46a8d9fe32.png)

Would you guys be interested in such a feature?

**Comment by ghost** [comment]
@tenebrius I totally agree with you. So your course table would look like this, right?

| Status | Title | Count (Lessons)
|---|---|---|
| • | Foo | 2
| · | Bar | 9
| • | Baz | 0

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
